Backend/BackendAPI/Repository/Repositories/GenericRepository.cs

```diff
@@ -31,7 +41,7 @@ public bool Insert(T component)
 
         public void SaveChanges()
         {
-            throw new NotImplementedException();
+            dbContext.SaveChanges();
         }
 
```

Comment:
tu trzeba łapać wyjątki zgodnie z dokumentacją https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.dbcontext.savechanges?view=efcore-2.1#Microsoft_EntityFrameworkCore_DbContext_SaveChanges

Reply:
Moge wyłapać wyjątki, ale nie jestem w stanie poinformować zewnętrznego świata o tym w żaden sposób .Jedyna rzecz to wrzucenie informacji do logów.

Reply:
Wydaje mi sie, ze lepiej bedzie aby wyjatek lecial na stos i informowal zewnetrzny swiat. Moge jedynie zdefiniowac wlasny wyjatek aby ulatwic proces.